src/sections/SMP/index.js

```diff
@@ -0,0 +1,140 @@
+                                <img src={latency} />
+                            </Col>                    
+                        </Row>
+                        <Row className="use-case-data">
+                            <Col className="use-case-box" lg={6}>
+                                <img src={client_capacity} />
+                            </Col>
+                            <Col className="use-case-box" lg={6}>
+                                <div>
+                                    <h1> Client Capacity </h1>
+                                    <p> Client Capacity figure also shows us that we are able to handle more requests than in the native case, although this infometric needs to be taken with a grain of salt, i.e. the difference might reduce if our application capacity was significantly larger than 100. </p>
+                                </div>
+                            </Col>
+                        </Row>
+                    </div>
+                </Container>
+                <div className="smp-effects">
+                    <Container>
+                        <div className="effects-title">
+                            <h1> Discreetly Studying the Effects of Individual Traffic Control Functions</h1>
+                            <h4> The group is also working in collaboration with the Envoy project to create easy-to-use tooling around distributed performance management (distributed load generation and analysis) in context of Istio, Consul, Tanzu Service Mesh, Network Service Mesh, App Mesh, Linkerd, and so on. </h4>
+                        </div>
+                    </Container>
+                    <div>
+                        {/* <Row> */}
+                        <iframe className="iframe" src="https://www.youtube.com/embed/rgnb0-ntPko"
+                            frameBorder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"allowFullScreen>
+                        </iframe>
+                        <iframe	className="iframe" src="https://www.youtube.com/embed/LxP-yHrKL4M"	
+                            frameBorder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"	allowFullScreen>	
+                        </iframe>
+                        <iframe	className="iframe" src="https://docs.google.com/presentation/d/e/2PACX-1vSWhREOZIfJqTG8jHtitgb5e-OAHC7E8EMmoVCaTRmC8b_itHkeFUNKcBVn9oaTpX5ozYvlhscvBXiK/embed?start=false&loop=false&delayms=3000"	
+                            frameBorder="0" allowFullScreen>	
+                        </iframe>	
+                        {/* </Row> */}
+                    </div>
+                </div>
+                <div className="cncf-callout">
+                    <h1> Join the CNCF Service Mesh Working Group </h1>
+                    <h4> Learn more about these initiatives</h4>
+                    <Button primary title="Learn More"></Button>
```

Comment:
@leecalcote What would be the link for this?

Reply:
If we don't have one or more links to https://smp-spec.io on the page already, then pointing them to the project site will suffice.

Reply:
We do have a button pointing to smp site.